var/spack/repos/builtin/packages/sos/package.py

```diff
@@ -81,6 +87,26 @@ def setup_dependent_build_environment(self, env, dependent_spec):
 
     def autoreconf(self, spec, prefix):
         bash = Executable("bash")
+        if spec.satisfies("@main") or spec.satisfies("@1.5.3:"):
+            branch_name = str(spec.version)
+            if spec.satisfies("@1.5.3:"):
+                branch_name = "v" + branch_name
+            try:
+                git = which("git")
```

Comment:
is it possible to have a dependence on tests-sos package instead of cloning

Reply:
Thinking more about this:  Perhaps [`spack.stage`](https://spack.readthedocs.io/en/latest/spack.html#spack.stage.Stage) could be used to download the `tests-sos` source.  Something like `stage.fetch`, `stage.expand_archive` into `./modules/tests-sos`.  Would this be preferred to `git('clone','...`

From my limited understanding of Spack, this would not be a formal `depends_on` relationship, because `sos` needs to build the `tests-sos` source from within its own directory tree.

Reply:
I don't know the best way to do it, but if it is only needed for a point release then it might not be worth putting a lot of time into it. Thanks for volunteering to do maintenance on the package. I will leave it to you. If you are new to spack package maintenance, then it would be good to join slack. There is a channel for packaging: https://spack.slack.com/